bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx102-de.html

```diff
@@ -0,0 +1,53 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_102_de') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-sm">
+      <div class="wnp-main-image mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-pocket">Pocket</div>
+      <h1 class="wnp-main-title">Lass dich inspirieren!</h1>
+      <p class="wnp-main-tagline">Erhalte die besten Artikel aus Pocket direkt in dein Postfach – jeden Tag!</p>
+
+      <p class="wnp-main-cta">
+        <a class="mzp-c-button mzp-t-product" href="https://getpocket.com/explore/pocket-hits-signup?utm_medium=mozilla-websites&utm_source=getpocket.com&utm_campaign=pocket-hits&utm_content=whatsnew-de">Melde dich für unseren Newsletter an</a>
```

Comment:
Is it worth adding some data attributes for GA here so we know how many people click the link?